### DISCUSSION TITLE
自定义命名空间+暗黑模式

### DISCUSSION THREAD

**Discussion by JieleeJie** [element-plus/element-plus#18835] [Q&A]
大大们，想问下，是不是使用自定义命名空间和暗黑模式没法同时启用，我看element-plus/theme-chalk/dark/css-vars.css中的类名是固定的el

**Comment by maplelost** [comment]
```
// we can add this to custom namespace, default is 'el'
//@forward 'element-plus/theme-chalk/src/mixins/config.scss' with (
//  $namespace: 'ep'
//);
```
for more information you can see https://github.com/element-plus/element-plus-vite-starter
